target/linux/layerscape/patches-4.9/601-net-support-layerscape.patch

```diff
@@ -2223,6 +447,15 @@ Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
  /* Make sure a field is enclosed inside headers_start/headers_end section */
  #define CHECK_SKB_FIELD(field) \
  	BUILD_BUG_ON(offsetof(struct sk_buff, field) <		\
+@@ -849,7 +875,7 @@ EXPORT_SYMBOL(napi_consume_skb);
+ 	BUILD_BUG_ON(offsetof(struct sk_buff, field) >		\
+ 		     offsetof(struct sk_buff, headers_end));	\
+ 
+-static void __copy_skb_header(struct sk_buff *new, const struct sk_buff *old)
++void __copy_skb_header(struct sk_buff *new, const struct sk_buff *old)
```

Comment:
Why did you add this and the above change?

Reply:
It appears I have refreshed on the wrong skbuff.c, the original patchset does drop the static but to copy_skb_header only. I will rebuild the patch and submit again.